### PR TITLE
Audit trail for monitors

### PR DESCRIPTION
**Pull request by SimoneT-DD**
### What does this PR do? What is the motivation?
We are replacing legacy audit events with Audit trail in the monitor status page. Because of this we need to update the public doc of the monitor status page to mirror this change.

### CODE REVIEW

File: content/en/monitors/manage/status.md
Comment (urseberry) [inline content/en/monitors/manage/status.md:138]:
```suggestion
Audit Trail automatically captures monitor changes for all monitor types and creates an event. This event documents the changes to the monitor.

 For example, in the case of an edit to a monitor, the Audit Trail event shows:
 - The previous monitor configuration
 - The current monitor configuration
 - The user that made the change
 
 For more information, see the [Audit Trail][11] documentation and read the [Audit Trail best practices][12] blog.
```

**Review by urseberry — CHANGES_REQUESTED** [review]
Thank you for keeping the public documentation up to date! I suggested some wording that I believe is easier to read.

**Comment by SimoneT-DD** [comment]
> Thank you for keeping the public documentation up to date! I suggested some wording that I believe is easier to read.

Thanks @urseberry 🙇‍♂️

**Review by urseberry — DISMISSED** [review]

File: content/en/monitors/manage/status.md
Comment (urseberry) [inline content/en/monitors/manage/status.md:182]:
```suggestion
[12]: https://www.datadoghq.com/blog/audit-trail-best-practices/
```
One last fix.

Also, after I approve, should I merge this PR?

Comment (SimoneT-DD) [reply content/en/monitors/manage/status.md]: Hi @urseberry,
we can merge it tomorrow when the change will be available in all the DC.

Thanks 🙇‍♂️

**Comment by SimoneT-DD** [comment]
Hi @urseberry,
we can merge this tomorrow when the change will be available in all the DC.

Thanks 🙇‍♂️

**Review by urseberry — APPROVED** [review]

**Pull request merged**
